pypeit/core/tracewave.py

```diff
@@ -4,27 +4,28 @@
 import copy
 
 import numpy as np
-import matplotlib.pyplot as plt
+import scipy
+from matplotlib import pyplot as plt
+from matplotlib import cm, lines
+
+from IPython import embed
+
+from astropy.stats import sigma_clipped_stats
 
 from pypeit import msgs
-from pypeit.core import arc
 from pypeit import utils
+from pypeit import tracepca
+from pypeit.core import arc
 from pypeit.core import qa
-from pypeit.core import trace_slits
-from pypeit.core import extract
-from astropy.stats import sigma_clipped_stats
-import matplotlib as mpl
-from matplotlib.lines import Line2D
-import scipy
-
-from IPython import embed
+from pypeit.core import trace
+from pypeit.core.moment import moment1d
 
+# TODO: Why is this try statement needed?
```

Comment:
we can get rid of it if we make ginga a requirement.
shall we?

Reply:
The main reason I asked was because there are lots of places in the code where we import `pypeit.ginga` without the try block.  So I think ginga is effectively a requirement already.  I'm removing the try block for this.